src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -1,130 +1,137 @@
 ---
 id: build-an-nft-marketplace
-title: Build an NFT Marketplace
-lastUpdated: 10th July 2023
+title: NFT Marketplace Part 1
+lastUpdated: 11th October 2023
 ---
 
 ## Introduction
 
-Business objects managed by a blockchain are called `assets`. On Tezos you will find the term `Financial Asset or FA` with different versions 1, 2, or 2.1.
+Welcome to the first part of our four-part series on building an NFT Marketplace. This tutorial aims to equip you with the knowledge and tools to create a robust NFT platform.
 
-Here are different categorizations of assets.
-
-![](http://jingculturecommerce.com/wp-content/uploads/2021/03/nft-assets-1024x614.jpg)
-
-## Wine marketplace
-
-We are going to build a Wine marketplace extending the `@ligo/fa` package from the [Ligo repository](https://packages.ligolang.org/). The goal is to showcase how to extend an existing smart contract and build a frontend on top of it.
-
-The Wine marketplace is adding these features on top of a generic NFT contract :
+In the first part, you learn:
 
-- mint new wine bottles
-- update wine bottle metadata details
-- buy wine bottles
-- sell wine bottles
+- The concepts of FA, IPFS, and smart contracts.
+- How to build an NFT Marketplace from the ligo/fa library.
 
-You can play with the [final demo](https://demo.winefactory.marigold.dev/).
-
-![nftfactory.png](/images/nftfactory.png)
+{% callout type="note" %}
+This training course is provided by [Marigold](https://www.marigold.dev/).
+You can find the 4 parts on github (solution + materials to build the UI)
 
-{% callout type="note" %} 
-Here we present Part 1 of 4 of a training course by [Marigold](https://www.marigold.dev/). You can find all 4 parts on github.
 - [NFT 1](https://github.com/marigold-dev/training-nft-1): use FA2 NFT template to understand the basics
 - [NFT 2](https://github.com/marigold-dev/training-nft-2): finish FA2 NFT marketplace to introduce sales
 - [NFT 3](https://github.com/marigold-dev/training-nft-3): use FA2 single asset template to build another kind of marketplace
 - [NFT 4](https://github.com/marigold-dev/training-nft-4): use FA2 multi asset template to build last complex kind of marketplace
-{% /callout %} 
+  {% /callout %}
```

Comment:
```suggestion
{% /callout %}
```

The indentation to this callout tag seems to be breaking the build.

Reply:
I pushed it ? Is it working now ?